lib/facebook/messenger/configuration/providers/environment.rb

```diff
@@ -27,8 +29,8 @@ def fetch_app_secret_proof_enabled?
             ENV['APP_SECRET_PROOF_ENABLED'] == 'true'.freeze
           end
 
-          def fb_graph_api_version
-            ENV['FB_GRAPH_API_VERSION']
+          def graph_api_version
+            GRAPH_API_VERSION
```

Comment:
Why are we defining this as a constant?

Reply:
This is just a measure to ensure that if a version isn't injected, there's a fallback in place.
In our case, we override this environment with `BotConfigurator`, but that weren't to happen, we would have this version to fallback to.